recipes/open62541/all/conanfile.py

```diff
@@ -329,6 +347,12 @@ def generate(self):
         # Honor BUILD_SHARED_LIBS from conan_toolchain (see https://github.com/conan-io/conan/issues/11840)
         tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
 
+        if version >= "1.4.11.1":
+            if self.settings.os == "Linux":
+                tc.cache_variables["UA_ENABLE_NODESETLOADER"] = self.options.get_safe("nodeset_loader", False)
+            else:
+                tc.cache_variables["UA_ENABLE_NODESETLOADER"] = False
```

Comment:
Pythonic and safer I guess 😉 
```suggestion
            tc.cache_variables["UA_ENABLE_NODESETLOADER"] = self.options.nodeset_loader if self.settings.os == "Linux" else False
```